lib/Thrift/Thrift.csproj

```diff
@@ -0,0 +1,12 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>net46;netstandard2.0</TargetFrameworks>
+    <TargetFrameworks Condition="$(OS) != 'Windows_NT'">netstandard2.0</TargetFrameworks>
```

Comment:
```suggestion
```

Reply:
Nitpicking here but I believe keeping the current value of `TargetFrameworks` and appending if condition is true makes a bit more sense:

```
  <PropertyGroup>
    <TargetFrameworks>netstandard2.0</TargetFrameworks>
  </PropertyGroup>
  <PropertyGroup Condition="'$(OS)' == 'Windows_NT'">
    <TargetFrameworks>net46;$(TargetFrameworks)</TargetFrameworks>
</PropertyGroup>
```

The reason being, when we add `netstandard2.1`, you just need to add to the top `TargetFrameworks`/add only once.

Although I'm not sure what's the reason to not compile for `net46` on non Windows. ReferenceAssemblies can be used for that. Tests can run on Mono.

Reply:
> > OK... figured out why I couldn't compile under windows. Seems like SourceLink isn't happy on my windows machine.
> 
> could you please share what is the problem? I don't have issues on my windows machine and I wonder if some prerequsites are missing from our contributor's guide.

I was getting this error:

`The "Microsoft.Build.Tasks.Git.LocateRepository" task failed unexpectedly.`

I can't give you specifics right now, since I just realized that I left my external drive that contains my windows image at home. I'll update this node with a more detailed error when I get home.

Reply:
here https://github.com/dotnet/sourcelink/issues/107 they tell "SourceLink requires .NET Core 2.1.300."

Do you have it installed on your machine? this version or later? Can you please check?

Reply:
> here [dotnet/sourcelink#107](https://github.com/dotnet/sourcelink/issues/107) they tell "SourceLink requires .NET Core 2.1.300."
> 
> Do you have it installed on your machine? this version or later? Can you please check?

Hi, yeah, I actually checked that the first thing when I switched over to Windows. 

I just finally figured it out. I didn't have Git installed on my Windows VM because I've been managing source control through my MacOS host. I didn't even catch this when tried to clone the repo, since I cloned it using Visual Studio 2019's "Clone or check out code" feature, and Visual Studio 2019 is able to clone without Git installed.

So, it may seem like a silly thing to add to the CONTRIBUTING.md, but we might want to add a line item to install the Git command line tools. Especially since Visual Studio 2019 can now interact with Git without the tools installed.